examples/src/loadES5.mjs

```diff
@@ -3,12 +3,13 @@
  * const CORE = await load('https://cdn.jsdelivr.net/npm/@loaders.gl/core@2.3.6/dist/dist.min.js');
  * const DRACO = await load('https://cdn.jsdelivr.net/npm/@loaders.gl/draco@2.3.6/dist/dist.min.js');
  * @param {string} url - The URL to ES5 file.
+ * @returns {Promise<Object>} - The module exports
  */
 export async function loadES5(url) {
     const res = await fetch(url);
     const txt = await res.text();
     const module = {
         exports: {}
     };
-    return (Function('module', 'exports', txt).call(module, module, module.exports), module).exports;
+    return (Function.prototype.constructor('module', 'exports', txt).call(module, module, module.exports), module).exports;
```

Comment:
Why do you make it more complicated than it is here, is it some ESLint rule?
![image](https://github.com/playcanvas/engine/assets/5236548/a60114c7-31ef-4ba9-af4b-3a6fddd18a18)

Reply:
This was just to be more explicit but I guess I can revert that lol.

Reply:
Also it bypasses the `eval` check for linting